### DISCUSSION TITLE
How to digitize reference shoreline

### DISCUSSION THREAD

**Discussion by Petronidas5** [kvos/CoastSat#510] [Q&A] [answered]
Hi everyone! I have been working lately with the Coastsat tool, and a vital question came to my mind when it comes to digitizing the reference shoreline to validate my results based on an image with higher spatial resolution. My doubt, therefore, is more of a conceptual question rather than a technical one, although it is quite relevant to the outcome of my validation.
I am aware, looking at the available literature, that the coastline is generally defined as the "sand/water interface," and this is where my doubt arises. In many cases (if not most), this interface is not clear when looking at a satellite image because there is wet sand with different degrees of moisture and, therefore, different shades of brown that make it very difficult to assign that line vector as the reference line at first glance.
I am attaching an image that exemplifies what I mean:
![PHR_image](https://github.com/kvos/CoastSat/assets/79979377/9f8892df-c94f-4e79-8237-046d633c61d4)

I guess that many of you have faced this at some point, so I hope to see if someone can shed some light on this matter. 
Thank you in advance!

**Comment by kvos** [comment, marked as answer]
yes that's the problem that we currently have when with sites that are mesotidal or macrotidal. In microtidal beaches you don't really have that problem as the portion of wet sand is very narrow. That's why for validation we use topographic surveys, rather than other imagery.
